### PR TITLE
Fix in test result selection and template improvement

### PR DESCRIPTION
**Pull request by gcomoretto**
I have fixed the selection of the test result, in case there is a NotExecute test result after the execution we have to report.

Also I the text in sections:
- 2.5 Related Documentation 
- 2.6 PMCS Activity

### CODE REVIEW

File: docsteady/templates/dm-tpr.latex.jinja2 [hunk 1]
```diff
@@ -205,9 +209,15 @@ Primavera milestones related to the test campaign.
 {% if testplan.pmcs_activity %}
 {{ testplan.pmcs_activity }}
 {% else %}
+  {% if testplan.milestone_id == "" %}
```
Comment (womullan) [inline docsteady/templates/dm-tpr.latex.jinja2:212]: The previous line should now  probably be  "Primavera milestones related to the test campaign:"  plus there is an extra blank line there it looks like

File: docsteady/templates/dm-tpr.latex.jinja2 [hunk 2]
```diff
@@ -195,6 +191,14 @@ To {{ cycle.key }} results &
 All documents provided as attachments in Jira are downloaded to Github and linked here for convenience.
 However, since they are not properly versioned, they should be considered informal and therefore
 not be part of the verification baseline.
+
+Consider using a Docushare collection to provide relevant documentation,
+instead of attaching documents to the Jira Test Plan {{ testplan.key }} and corresponding Test Cycle(s).
+{% else %}
+No additional documentation provided.
+Consider using a Docushare collection to provide relevant documentation,
```
Comment (womullan) [inline docsteady/templates/dm-tpr.latex.jinja2:199]: this needs to be communicated to the person writing the test not someone reading the report..

**Review by womullan — CHANGES_REQUESTED** [review]
this looks good - but 2.5 is too instructional .

File: docsteady/templates/dm-tpr.latex.jinja2 (hunk 2 above)
Comment (gcomoretto) [reply docsteady/templates/dm-tpr.latex.jinja2:199]: OK, I remove this text.

**Review by womullan — APPROVED** [review]
ok

**Pull request merged**